deploy/FlapperInit.sol

```diff
@@ -135,26 +125,39 @@ library FlapperInit {
     }
 
     function initSplitter(        
-        DssInstance memory dss,
-        address splitter,
-        SplitterConfig memory cfg
+        DssInstance      memory dss,
+        SplitterInstance memory splitterInstance,
+        SplitterConfig   memory cfg
     ) internal {
+        SplitterLike    splitter = SplitterLike(splitterInstance.splitter);
+        SplitterMomLike mom      = SplitterMomLike(splitterInstance.mom);
+
         // Sanity checks
-        SplitterLike splitter_ = SplitterLike(splitter);
-        require(splitter_.vat()     == address(dss.vat),     "Splitter vat mismatch");
-        require(splitter_.daiJoin() == address(dss.daiJoin), "Splitter daiJoin mismatch");
+        require(splitter.vat()     == address(dss.vat),     "Splitter vat mismatch");
+        require(splitter.daiJoin() == address(dss.daiJoin), "Splitter daiJoin mismatch");
+        require(splitter.farm()    == cfg.farm,             "Splitter farm mismatch");
 
-        require(cfg.burn <= WAD, "Splitter burn too high");
+        require(cfg.hump > 0,         "hump too low");
+        require(cfg.hop >= 5 minutes, "hop too low");
+        require(cfg.burn <= WAD,      "burn too high");
+
+        splitter.file("hop",  cfg.hop);
+        splitter.file("burn", cfg.burn);
+        splitter.rely(address(mom));
 
-        splitter_.file("burn", cfg.burn);
-        FarmLike farm = FarmLike(splitter_.farm());
-        farm.setRewardsDistribution(splitter);
+        FarmLike farm = FarmLike(cfg.farm);
+        farm.setRewardsDistribution(splitterInstance.splitter);
         farm.setRewardsDuration(cfg.rewardsDuration);
 
-        // Wire splitter with vow
```

Comment:
Might be nicer to keep this comment, as otherwise it's a bit unclear why these configs are separated from the splitter chunk above and the vow chunk below.

Reply:
Regrouped things in 79a0021c